### DISCUSSION TITLE
Invalid cluster.id When Reinstalling Kafka with Existing PVCs

### DISCUSSION THREAD

**Discussion by willyaako** [strimzi/strimzi-kafka-operator#10851] [Q&A]
When using Strimzi Kafka in KRAFT, I set the deleteClaim option for Kafka nodes to **false**. 
After removing the entire Kafka cluster, the PVCs remain intact.

However, when I try to reinstall the same Kafka cluster, I encounter the following error:

```
Invalid cluster.id in: /var/lib/kafka/data-0/kafka-log3/meta.properties. Expected {NEW_CLUSTER_ID}, but read {OLD_CLUSTER_ID}
```

What solutions are there for this issue?
Any guidance would be greatly appreciated. Thank you!

**Comment by scholzj** [comment]
That is because you are trying to recover the cluster from the existing PVCs. If that is what you want to do, you have to first restore the cluster ID. If what you want is a brand new Kafka cluster, you have to delete the PVCs after you delete the Kafka cluster.

**Comment by fvaleri** [comment]
Hi @willyaako, points 8-11 of [this procedure](https://strimzi.io/docs/operators/latest/deploying#proc-cluster-recovery-volume-str) may help with restoring the `cluster.id`. Note that this is an expert procedure, and not exactly a match with your use case, so be careful.
